app/src/renderer/components/governance/PageProposal.vue

```diff
@@ -101,7 +97,25 @@ export default {
   }),
   computed: {
     // TODO: get denom from governance params
-    ...mapGetters([`bondingDenom`]),
+    ...mapGetters([`bondingDenom`, `proposals`]),
+    proposal() {
+      let proposal = this.proposals[this.proposalId]
+      if (proposal) {
+        proposal.tally_result.yes = String(
+          Math.round(parseFloat(proposal.tally_result.yes))
+        )
+        proposal.tally_result.no = String(
+          Math.round(parseFloat(proposal.tally_result.no))
+        )
+        proposal.tally_result.no_with_veto = String(
```

Comment:
this is weird, no? does it have to be a string?

`String(Math.round(parseFloat(somethingHere)))`

Reply:
well not necessarily

Reply:
fixed